.ci/pytorch/macos-test.sh

```diff
@@ -46,7 +46,7 @@ test_python_shard() {
 
   setup_test_python
 
-  time python test/run_test.py --verbose --exclude-jit-executor --exclude-distributed-tests --shard "$1" "$NUM_TEST_SHARDS"
+  time python test/run_test.py --verbose --exclude-jit-executor --exclude-distributed-tests --exclude-inductor-codegen-tests --shard "$1" "$NUM_TEST_SHARDS"
```

Comment:
Shouldn't we default these tests as excluded, if we expect them to be failing most of the time?

Reply:
If we decide to go with unstable, OK. But see my reply below. I think we should just enable this as part of ordinary workflows -- no CI changes would be necessary, too.